male-seed-beetle/scripts/convert-with-core.py

```diff
@@ -1,7 +1,57 @@
 import polars as pl
+import re
+from pathlib import Path
+
+#Transform from tab sep to comma sep
+with open('../data-raw/data.csv', 'r') as tsv:
+
+    with open('../data-raw/data1.csv', 'w') as csv:
+
+        for line in tsv:
+
+            content = re.sub("\t", ",", line) 
+
+            csv.write(content)
```

Comment:
Include option to overwrite if it isn't already the default.

Reply:
It'll run multiple times in a row, so it looks like it is default behaviour.